build_mmdeploy.sh

```diff
@@ -0,0 +1,346 @@
+    # torchvision
+    sudo apt-get install libjpeg-dev zlib1g-dev libpython3-dev libavcodec-dev libavformat-dev libswscale-dev -y
+    sudo rm -r torchvision
+    git clone --branch v0.11.1 https://github.com/pytorch/vision torchvision
+    cd torchvision
+    export BUILD_VERSION=0.11.1  # where 0.x.0 is the torchvision version
+    python3 setup.py install
+    cd ../
+    # torchaudio
+    #sudo apt-get install -y sox libsox-dev libsox-fmt-all
+    #sudo rm -r torchaudio
+    #git clone -b v0.10.0 https://github.com/pytorch/audio torchaudio
+    #cd torchaudio
+    #git submodule update --init --recursive
+    #python3 setup.py install
+    #cd ../
+    # mmcv
+    pip3 uninstall mmcv-full
+    pip3 install mmcv-full==1.4.1 -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.10.0/index.html
+  else
+    pip3 install torch==1.10.0+cu113 torchvision==0.11.1+cu113 torchaudio==0.10.0+cu113 -f https://download.pytorch.org/whl/cu113/torch_stable.html
+    # mmcv
+    pip3 uninstall mmcv-full
+    pip3 install mmcv-full==1.4.1 -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.10.0/index.html
+  fi
+
+}
+
+pplcv() {
+  ## ppl.cv - install in /usr/local
+  cd ${WORKING_DIR}
+  echo_blue "checking out '${PPLCV_DIR}' pkg..."
+  if [ -d "${PPLCV_DIR}" ]; then
+      echo_green "Already exists! Checking out the requested version..."
+  else
+      git clone https://github.com/openppl-public/ppl.cv.git ${PPLCV_DIR}
+  fi
+  cd ${PPLCV_DIR}
+  git pull
+  git checkout tags/v${PPLCV_VER}
+  mkdir build -p && cd build
+  cmake -DHPCC_USE_CUDA=ON -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} .. && make -j${processor_num} && sudo make install
+  sudo ldconfig
+  # pack as tar.gz file
+  cd ..
+  tar -zcvf ${WORKING_DIR}/pplcv_${PPLCV_VER}_cuda-${ARCH}-build.tar.gz build/
+}
+
+mmdeploy(){
+  ## h5py (Required by mmdeploy)
+  ## h5py not directly supported by jetson and must be built/installed manually
+  sudo apt-get install pkg-config libhdf5-10* libhdf5-dev -y
+  sudo pip3 install Cython
+  sudo env H5PY_SETUP_REQUIRES=0 pip3 install -U h5py==2.9.0
+
+  ## mmdeploy SDK - install in /usr/local
+  cd ${WORKING_DIR}
+  echo_blue "checking out '${MMDEPLOY_DIR}' pkg..."
+  if [ -d "${MMDEPLOY_DIR}" ]; then
+    echo_green "Already exists! Checking out the requested version..."
+  else
+    git clone https://github.com/open-mmlab/mmdeploy.git ${MMDEPLOY_DIR}
+  fi
+  cd ${MMDEPLOY_DIR}
+  git pull
+  git checkout tags/v${MMDEPLOY_VER}
+  # reinit submodules
+  git submodule update --init --recursive
+
+  if [[ $WITH_PYTHON -eq 1 ]]
+  then
+    pip install -e .
+  fi
+  rm -r build
+  mkdir build -p && cd build
+  cmake .. \
+    -DMMDEPLOY_BUILD_SDK=ON \
+    -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
+    -DCMAKE_CXX_COMPILER=g++ \
+    -Dpplcv_DIR=/usr/local/lib/cmake/ppl \
+    -DMMDEPLOY_TARGET_DEVICES="cuda;cpu" \
+    -DMMDEPLOY_TARGET_BACKENDS=trt \
+    -DMMDEPLOY_CODEBASES=all \
+    -DTENSORRT_DIR=${TENSORRT_DIR} \
+    -DCUDNN_DIR=${CUDNN_DIR}
```

Comment:
Please add `-DMMDEPLOY_BUILD_SDK_PYTHON_API=ON` so as to build SDK's python package